extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -578,7 +578,7 @@ export class GmailElementReplacer implements WebmailElementReplacer {
   };
 
   private getConvoRootEl = (anyInnerElement: HTMLElement) => {
-    return $(anyInnerElement).closest('div.if, td.Bu').first();
+    return $(anyInnerElement).closest('div.Bk, div.if, td.Bu').first();
```

Comment:
I've just noticed that it'll be better to use `div.aHU` selector, as it's used by root conversation element, while `div.Bk` is used by each message

Reply:
I just merged the hotfix from the `master` branch.